src/frontend/src/styles/main.css

```diff
@@ -2158,6 +2180,80 @@ input[type="checkbox"] {
   image-rendering: pixelated;
 }
 
+/* Marquee */
+
+.c-marquee {
+  --marqueeGap: var(--rs-marquee-gap);
+
+  position: relative;
+  overflow: hidden;
+  height: 40rem;
+}
+
+.c-marquee__row {
+  position: absolute;
+  top: calc(var(--rowIndex) / var(--totalRows) * 100% + var(--marqueeGap));
+  left: 0;
+  width: calc(200% + var(--marqueeGap) * var(--itemsPerRow) * 2);
+  height: calc(100% / var(--totalRows) + var(--marqueeGap));
+  align-items: center;
+  justify-content: center;
```

Comment:
Should `c-marquee__row` also have `display: flex`?

Reply:
good catch, I was able to remove these, since the logos are now positioned absolutely